### PR TITLE
Travis CI: Swap gcc-7 for gcc-9

### PR DESCRIPTION
**Pull request by Headline**
GCC 7.0 always failed with macro redefinition errors (beyond our control, anyway), so might as well swap for GCC 9.x

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,8 +31,8 @@ addons:
       - g++-4.9
       - g++-5-multilib
       - g++-5
-      - g++-7-multilib
-      - g++-7
+      - g++-9-multilib
+      - g++-9
```
Comment (KyleSanderson) [inline .travis.yml]: hmmm. Looks like SM doesn't actually build with gcc-9...

Comment (Headline) [reply .travis.yml]: hmm, mine built ok, but this is quite sad

Comment (Kenzzer) [reply .travis.yml]: image
Checks passed though?

**Comment by Headline** [comment]
Closed since gcc-9 cannot successfully build sourcemod with all hl2sdks unless further invasive intervention occurs, which is not justifiable at this time

**Pull request closed without merging**